braindecode/datautil/windowers.py

```diff
@@ -18,15 +20,17 @@
 def create_windows_from_events(
         concat_ds, trial_start_offset_samples, trial_stop_offset_samples,
         window_size_samples=None, window_stride_samples=None,
-        drop_last_window=False,
-        mapping=None, preload=False, drop_bad_windows=True):
-    """Windower that creates windows based on events in mne.Raw.
-
-    The function fits windows of window_size_samples in
-    trial_start_offset_samples to trial_stop_offset_samples separated by
-    window_stride_samples. If the last window does not end
-    at trial_stop_offset_samples, it creates another overlapping window that
-    ends at trial_stop_offset_samples if drop_last_window is set to False.
+        drop_last_window=False, mapping=None, preload=False,
+        drop_bad_windows=True, picks=None, reject=None, flat=None,
+        on_missing='error'):
+    """Create windows based on events in mne.Raw.
+
+    This function extracts windows of size window_size_samples in the interval
+    [trial_start_offset_samples, trial_stop_offset_samples] around each event,
+    with a separation of window_stride_samples between consecutive windows. If
+    the last window around an event does not end at trial_stop_offset_samples
+    and drop_last_window is set to False, an additional overlapping window that
+    ends at trial_stop_offset_samples is created.
 
     in mne: tmin (s)                    trial onset        onset + duration (s)
     trial:  |--------------------------------|--------------------------------|
```

Comment:
is doc here maybe wrong/inconsistent? 
Below it says:
` trial_stop_offset_samples -       Stop offset from original trial stop, in samples.`
This is also what happens in code in _compute_window_inds:
https://github.com/braindecode/braindecode/blob/fac6cea5d6ea0af49dcb39c85d3be33dfa791c22/braindecode/datautil/windowers.py#L290-L294

Reply:
Good point, I'll fix the docstring.